newsfragments/2727.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix large-scale tests for the new Umbral API
```

Comment:
Don't think that we need a newsfragment for fixing of tests.

Reply:
Removed.